src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateCollectionCommand.java

```diff
@@ -137,7 +137,7 @@ public record VectorizeConfig(
           @Nullable
               @Schema(
                   description =
-                      "Optional parameters that match the template provided for the provider",
+                      "Optional parameters that match the messageTemplate provided for the provider",
```

Comment:
@amorton Is this intentional change or bulk-replace?

Reply:
I assume bulk replace since there's no `messageTemplate` around here, will revert

Reply:
I assume it is intentional as there's further changes in `EmbeddingProvider`.